<launch>
    <arg name="config_file" default="sam.yaml"/>
    <arg name ="robot_name" default ="sam_auv_v1"/>
    <arg name="use_motion_model" default="true"/>
    <arg name="inference_strategy" default="FullSmoothing"/> 
    <node pkg="hydrobatic_localization" exec="state_estimator" output="screen" namespace = "$(var robot_name)">
        <param name = "config_file" value = "$(var config_file)"/>
        <param name = "use_sim_time" value = "true"/>
        <param name = "use_motion_model" value = "$(var use_motion_model)"/>
        <param name = "inference_strategy" value = '$(var inference_strategy)'/>
        <param name = "init_from_ground_truth" value = "false"/>
    </node>


</launch>
